package.json

```diff
@@ -61,19 +61,21 @@
     "ajv-formats": "^2.1.1",
     "chai": "^4.3.4",
     "mocha": "^9.1.1",
+    "npm-run-all": "^4.1.5",
     "prettier": "^2.4.1",
     "source-map-support": "^0.5.20",
     "tslint": "^6.1.3"
   },
   "scripts": {
-    "prepublishOnly": "yarn build",
-    "test": "yarn build && mocha -t 5000 --require source-map-support/register dist/test",
+    "prepare": "run-s build",
+    "test": "run-s build test:mocha",
```

Comment:
I don't think we need npm compat and I'd rather not add the dependency.

Reply:
This dependency is widely used and only as dev, so it does not cost anything. Using the `prepare` script and run-s instead of yarn means that github forks installs will work and especially if users use npm (most do).

Reply:
And also using `prepare` is what npm [suggests](https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish).

Reply:
Since we use yarn.lock, I don't want developers to use npm. Using prepare sounds good to me.

Reply:
I'm going to open a new pull request without the dependency

Reply:
Can you update this pull request instead?

Reply:
I'm sorry I did not know how to do so, I deleted this branch and created a new pr